### ISSUE TITLE
AlwaysShow and classes

### ISSUE THREAD

**Issue by gquipster** [mimo84/bootstrap-maxlength#102]
Hi,

If you use the alwaysShow property to display the counter whenever the input is selected, it by default uses the warning class. Is it possible to add an extra property so I can have:

Label is green when 0 -> threshold characters are present
Label is amber when threshold -> max characters are present
Label is red when max characters are present

Thanks

**Comment by mikelkew** [comment]
I agree, this would be the ideal behavior. Perhaps using a defaultClass option that is used until threshold is reached.

**Comment by davdenic** [comment]
I vote this feature! 👍

**Comment by qwe0001** [comment]
Hey, does anyone have any news on this?
I want this feature really.

My working project won't wait for this feature to be implemented,
so I've implemented a temporary solution.

You can see it in the demo below.

https://codepen.io/qwe001/pen/LYGYxMa

I hope something close to this will be implemented as an official option!
